CoastalResilienceNetwork/GeositeFramework
# Remove layer selector v2 plugin

* This plugin has been moved to
https://github.com/CoastalResilienceNetwork/regional-planning.
* Future development will happen there.

**Testing**
- Verify there are no JS errors related to the removal of the new layer selector.
- Verify the legend still works correctly with the old layer selector.

Connects to #613
